resources/assets/js/components/DashboardRightSidebar.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div>
+    <div class="d-none d-md-block">
+      <b-img-lazy fluid src="/images/dashboard_3.jpg" class="border border-dark border-bottom-0" />
+      <div class="pt-4 pl-4 pr-4 pb-2 greyish border border-dark border-bottom-0">
+        <p class="font-weight-bold">
+          <!--        eslint-disable-next-line-->
+          {{ translatedSidebarIntro1 }}
+          <span class="d-inline d-md-none">
+            {{ translatedSidebarKit1 }} <a href="https://talk.restarters.net/t/how-to-run-a-repair-event-the-restart-party-kit/324">{{ translatedSidebarKit2 }}</a>
```

Comment:
We should review whether we need to use the special links that force login to Discourse via SSO, if not already logged in - added to retro list.

Reply:
Following discussion, we said yes we would keep them on pages that are behind login.